src/RpcServer/RpcServer.cs

```diff
@@ -55,7 +57,22 @@ private bool CheckAuth(HttpContext context)
 
         protected override void Configure()
         {
-            Settings.Load(GetConfiguration());
+            IConfigurationSection config = null;
+            int remainingTimes = 3;
+            while (remainingTimes > 0 && config == null)
+            {
+                try
+                {
+                    config = GetConfiguration();
+                }
+                catch (FormatException)
+                {
+                    remainingTimes--;
+                    Thread.Sleep(10);
+                }
+            }
```

Comment:
I think we can just catch the exception and ignore it.

Reply:
If we ignore the exception without try once more, a null config will be loaded, which will give rise to a null-pointer exception and crash the node

Reply:
If you don't invoke `Settings.Load()`, then it won't.

Reply:
Got it, have changed correspondingly